virttest/qemu_vm.py

```diff
@@ -4860,6 +4860,12 @@ def _set_migrate_capability(vm, capability, value, is_src_vm=True):
                         if os.path.isfile(save2):
                             os.remove(save2)
 
+            # FIXME: Never remove the source VM's files when doing a local
+            # migration because these files are still being used by the
+            # target VM, let the target VM do the cleanup work
+            if local and self.devices.temporary_image_snapshots:
+                self.devices.temporary_image_snapshots.clear()
```

Comment:
The dst VM will generate the **temporary_image_snapshots** but not copy from the src VM, even if they are the same, not sure if there is a potential problem. Ack this time.